src/main/java/com/diffblue/javademo/tictactoe/Board.java

```diff
@@ -0,0 +1,83 @@
+package com.diffblue.javademo.tictactoe;
+
+
+public class Board {
+
+  private Player[][] board;
+
+  /**
+   * Initialize board.
+   *
+   * @param size row
+   */
+  public Board(int size) {
+    // Check if the board is TicTacToe playable
+    if (size % 2 == 0 || size <= 1) {
+      throw new IllegalArgumentException("The board is not suited to play TicTacToe.");
```

Comment:
Please could you update the error to follow the format:
1) Tell the user what they did wrong
2) Tell the user what the correct input is